src/runtime/builder/mod.rs

```diff
@@ -3,15 +3,12 @@ use wasmtime::{
     component::{Linker, ResourceTable},
     Config, Engine, Store,
 };
-use wasmtime_wasi::preview2::{
-    command::{self},
-    StdinStream, StdoutStream, WasiCtxBuilder,
-};
+use wasmtime_wasi::{StdinStream, StdoutStream, WasiCtxBuilder};
```

Comment:
It seems that there are no `preview2` and `command` anymore. Does that mean WASI run into stable status?

Reply:
According to this pr: https://github.com/bytecodealliance/wasmtime/pull/7933

preview2 have been merged in to root mod. so there are no more preview2, Command have been moved too. 

However I don't know if this means WASI have reached stabled status.